src/server/system_services/pool_server.js

```diff
@@ -1215,12 +995,8 @@ function calc_hosts_pool_mode(pool_info, is_initialized, storage_by_mode, s3_by_
         .reduce((sum, val) => sum + val.count, 0);
     const activity_ratio = (activity_count / host_count) * 100;
 
-    const now = Date.now();
-    const init_timeout = system_store.data.systems[0].pools_by_name[pool_info.name].hosts_pool_info.init_timeout || now;
-
     return (is_managed && hosts.configured_count === 0 && 'DELETING') ||
-        (is_managed && !is_initialized && (now > init_timeout) && 'INITIALIZING_FAILED') ||
-        (is_managed && !is_initialized && 'INITIALIZING') ||
+        (is_managed && host_count === hosts_initializing && 'INITIALIZING') ||
```

Comment:
This is not always correct, what if the operator should bring up only half of the pod (which means half of the agants) like in the case where it has no nodes to allocate to the pods. In this case the pool should still be in initializing state BUT only if this is the operation that started in the creation (this is why we had the initialize flag on the host_info, that you removed).

In the case of scaling it is ok because the pool might already be in use

Reply:
sounds better to me. Initializing was always a temporary step before nodes started to connect. if some has connected we will move to scaling as we are trying to get to the desired state but you can start working. Init-failed i this case sound harsh.